pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -888,8 +888,8 @@ func (ctrl *Controller) syncImageConfig(key string) error {
 		if applied {
 			klog.Infof("Applied ImageConfig cluster on MachineConfigPool %v", pool.Name)
 		}
+		ctrlcommon.UpdateStateMetric(ctrlcommon.MCCSubControllerState, "machine-config-controller-container-runtime-config", "Sync Image Config", pool.Name)
```

Comment:
Assuming I understood it correctly, I think the containerruntimeconfig sync metric only happens if an actual change happened.

Should we do the same here? i.e. move this metric into the `if applied` block?